### DISCUSSION TITLE
Make tabs open by clicking the middle mouse. Open a tab in the opposite panel "Ctrl+Shift+Up". "Ctrl+Left/Right Arrow" to move through words. Ability to select a path in the address bar using left-click

### DISCUSSION THREAD

**Discussion by moninahh** [doublecmd/doublecmd#1851] [Suggestions]
1. Make tabs open by clicking the middle mouse button on an empty space on the tab bar. This function is missing. It is automatically more comfortable and convenient when the tab is opened and closed by middle click.

2. Open a tab in the opposite panel "Ctrl+Shift+Up".

3. When creating a file "Shift+F4", the ability to use "Ctrl+Left/Right Arrow" to move through words.

4. Ability to select a path in the address bar using left-click. Not by right-clicking.

**Comment by moninahh** [comment]
5. For the "cm_TransferLeft" and "cm_TransferRight" commands, add a parameter to load a directory in the target panel without navigating into it.

6. Create a parameter for the "cm_TargetEqualSource" and "cm_NewTab" commands that makes them navigate into the directory under the cursor (similar to the behavior of "cm_OpenDirInNewTab", "cm_TransferLeft", and "cm_TransferRight").

7. In the Favorite Tabs and Favorite Directories configuration, make it possible to select all items using "Ctrl+A".

**Comment by ags1234** [comment]
8. Make the option "Move to archive" in Alt+F5 dialog actually work for zip archives.

**Comment by JonnyTech** [comment]
Open a seperate issue for each feature request, please.
